rdkit/Chem/Draw/MolDrawing.py

```diff
@@ -231,14 +210,15 @@ def _drawWedgedBond(self, bond, pos, nbrPos, width=None, color=None, dash=None):
 
   def _drawBond(self, bond, atom, nbr, pos, nbrPos, conf, width=None, color=None, color2=None,
                 labelSize1=None, labelSize2=None):
-    if width is None:
-      width = self.drawingOptions.bondLineWidth
-    if color is None:
-      color = self.drawingOptions.defaultColor
+    width = width or self.drawingOptions.bondLineWidth
+    color = color or self.drawingOptions.defaultColor
+    color2 = color2 or self.drawingOptions.defaultColor
```

Comment:
The handling of color2 is further down in the original code, but makes more sense to have here.